diff --git a/node_modules/hexo-asset-image/index.js b/node_modules/hexo-asset-image/index.js
index d651afa..7571ee6 100644
--- a/node_modules/hexo-asset-image/index.js
+++ b/node_modules/hexo-asset-image/index.js
@@ -21,7 +21,7 @@ hexo.extend.filter.register('after_post_render', function(data){
       var endPos = link.lastIndexOf('/');
     }
     else {
-      var endPos = link.lastIndexOf('.');
+      var endPos = link.lastIndexOf('/');
     }
     link = link.substring(beginPos, endPos) + '/' + appendLink;
 
